package.json

```diff
@@ -10,7 +10,7 @@
     "clean:build": "lerna run clean --stream",
     "clean:modules": "lerna clean --yes",
     "clean": "npm run clean:build && npm run clean:modules",
-    "bootstrap": "lerna bootstrap --hoist -- --no-audit --prefer-offline",
+    "bootstrap": "lerna bootstrap --hoist --nohoist=bootstrap -- --no-audit --prefer-offline",
```

Comment:
what's the purpose of this?

Reply:
`custom.scss` imports from the Bootstrap version in node_modules. Since we're shipping that scss file to enterprise without compiling it, it still needs to be able to import Bootstrap. Sass has no real module resolution, so in OSS the import would be 3 or 4 levels up, but in enterprise it would be just 2 levels up if bootstrap were also installed to enterprise.

Nohoist emulates the package structure (with regular dependencies) when installed via npm. So importing `../node_modules/boostrap/...` will work in OSS and enterprise as both will have bootstrap in the components node_modules folder